index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/4339e938ba.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>my portofolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <header class="logo">Amanuel</header>
+        <a href="#" class="humburger"><i class="fa-solid fa-bars"></i></a>
+      </nav>
+      <main class="main-content">
+        <h1 class="heading1">Hey There.<br />I’m Amanuel</h1>
+        <h3 class="heading3">I am Software Engineer.</h3>
+        <p class="myinfo">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-icons">
+          <li>
+            <a href="#"><i class="fa-brands fa-github"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-angellist"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-solid fa-m"></i></a>
+          </li>
+        </ul>
```

Comment:
Please put those elements inside a `section` tag because It should represent a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.